src/app/games/rectanglz/RectanglzTutorial.ts

```diff
@@ -0,0 +1,114 @@
+const initialState: RectanglzState = RectanglzRules.get().getInitialState(defaultConfig);
+
+const _: PlayerOrNone = PlayerOrNone.NONE;
+const O: PlayerOrNone = PlayerOrNone.ZERO;
+const X: PlayerOrNone = PlayerOrNone.ONE;
+
+export class RectanglzTutorial extends Tutorial {
+
+    public tutorial: TutorialStep[] = [
+
+        TutorialStep.informational(
+            $localize`Rectanglz`,
+            $localize`Rectanglz is board control game. Here is the initial state. The goal is to have the more piece at the end of the game.`,
+            initialState,
+        ),
+
+        TutorialStep.fromPredicate(
+            $localize`Simple step`,
+            $localize`One of the two kind of move you can do is the simple step. When you do one, you create a new piece. To do this, select one of your pieces, and click on its neighbors space.<br/>You're playing Dark, make a simple step.`,
+            initialState,
+            RectanglzMove.from(new Coord(0, 0), new Coord(1, 1)).get(),
+            (move: RectanglzMove, _: RectanglzState) => {
+                if (move.isDuplication()) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure($localize`This was a single step, try a jump now.`);
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ),
+
+        TutorialStep.fromPredicate(
+            $localize`Jumps`,
+            $localize`The second type of move you can do is the jump. When you do one, your piece leaves it's original space and jump two space further. To do this, select one of your pieces, and click on its landing space two space further.<br/>You're playing Light, make a jump.`,
+            new RectanglzState([
+                [O, _, _, _, _, _, _, X],
+                [_, O, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _],
+                [X, _, _, _, _, _, _, O],
+            ], 1),
+            RectanglzMove.from(new Coord(7, 0), new Coord(5, 2)).get(),
+            (move: RectanglzMove, _: RectanglzState) => {
+                if (move.isDuplication()) {
+                    return MGPValidation.failure($localize`This was a single step, try a jump now.`);
+                } else {
+                    return MGPValidation.SUCCESS;
+                }
+            },
+            TutorialStepMessage.CONGRATULATIONS(),
+        ).withPreviousMove(RectanglzMove.from(new Coord(0, 0), new Coord(1, 1)).get()), // TODO: check that we SEE the highlight
+        TutorialStep.fromPredicate(
+            $localize`Captures`,
+            $localize`When one of your pieces arrives on a square neighbor to opponent's pieces, those pieces switch color.<br/>You're playing Dark, do such a move!`,
```

Comment:
arrives -> lands (ou "lands or is duplicated on")
on a square that neighbors* ou on a neighbor square of* 
"these pieces become yours" plutôt pour emphaser ?

Reply:
mal corrigé ! t'as écrit "on a space neighbors"

Reply:
please no more comments under outdated code ! next comments will be considered as wrong